src/ResourceManager/ServiceBus/Commands.ServiceBus/Cmdlets/AzureServiceBusCmdletBase.cs

```diff
@@ -94,12 +94,16 @@ public abstract class AzureServiceBusCmdletBase : AzureRMCmdlet
         protected const string AliasAuthRuleObj = "AuthRuleObj";
         protected const string AliasSubscriptionName = "SubscriptionName";
         protected const string AliasSubscriptionObj = "SubscriptionObj";
-
+        protected const string AliasAliasName = "Alias";
 
         protected const string ServicebusSubscriptionVerb = "AzureRmServiceBusSubscription";
 
         protected const string ServicebusRuleVerb = "AzureRmServiceBusRule";
 
+        protected const string ServicebusDRConfigurationVerb = "AzureRmServiceBusDRConfigurations";
```

Comment:
@v-Ajnava cmdlet nouns should be singular, so this should be `Configuration` rather than `Configurations`

Reply:
Resolved